test/testdrive/avro-cdcv2.td

```diff
@@ -123,12 +123,12 @@ $ kafka-ingest format=avro topic=data schema=${schema} timestamp=3
 # that restarted and emits previously emitted data at a compacted timestamp
 
 $ kafka-ingest format=avro topic=data schema=${schema} timestamp=1
-{"array":[{"data":{"id":5,"price":{"int":10}},"time":9,"diff":1}]}
-{"array":[{"data":{"id":5,"price":{"int":12}},"time":9,"diff":1}]}
-{"array":[{"data":{"id":5,"price":{"int":12}},"time":9,"diff":-1}]}
+{"array":[{"data":{"id":5,"price":{"int":10}},"time":5,"diff":1}]}
```

Comment:
To preserve the spirit of the test, we could emit these at times `9`, `8`, and `9`, and adapt the progress statement below to still be from `0` to `10`. The idea was to test the scenario where a producer re-produces data, but at a compacted timestamp. And CDCv2 should filter these out succesfully.

Reply:
@aljoscha it seems like we resolved this on slack but lmk if you still need me to make changes?